### PR TITLE
strategies show on know.html 

### PR DESCRIPTION
**Pull request by Zmuse001**
### Changes
styled the students know and show.html to look like the screenshot

### Link to Issue 
[Link Asana ticket here!](https://app.asana.com/0/1206510424537042/1206645498622461/f)

### Comments, Notes, or  Questions
accidentally created 2 branches in 1 and had to recreate the branch from scratch.

### Related Screenshots 
For FE tickets specifcially, please include a screenshot of your work!
Screenshot 2024-05-15 at 3 07 47 PM

### Before you submit for review, Did you...

- [ ] Added relevant tests (if applicable)
- [ ] Linked Asana ticket to PR
- [ ] Added comments, notes, or questions for reviewwer
- [ ] Added relevant screenshots
- [ ] Ensured ticket title is clear and concise(i.e. 'Donation Form Feature' instead of 'sfg/chels/donation-form'

### CODE REVIEW

**Review by crestu — APPROVED** [review]
styling looks good! nice job

File: app/views/layouts/_sidebar_nav.html.erb [hunk 1]
```diff
@@ -12,6 +12,7 @@
           <%= link_to students_path, class: "nav-tab" do %>
             <!--add my_students icon here-->
             <img src="<%= asset_path("new_sidebar_icons/my_students_icon.png") %>" alt="my_students_icon" class="h-24 px-5"/>
+            <i class="fa-brands fa-font-awesome"></i>
```
Comment (edithemmings) [inline app/views/layouts/_sidebar_nav.html.erb:15]: ^Why add this icon to the sidebar?

File: app/views/way_of_being/index.html.erb
Comment (edithemmings) [inline app/views/way_of_being/index.html.erb:1]: I don't think you want to be deleting the way of being index

File: app/views/layouts/_sidebar_nav.html.erb (hunk 1 above)
Comment (Zmuse001) [reply app/views/layouts/_sidebar_nav.html.erb:15]: fixed it.

File: app/assets/images/Icon.svg [hunk 2]
```diff
@@ -1,3 +0,0 @@
-<svg width="16" height="12" viewBox="0 0 16 12" fill="none" xmlns="http://www.w3.org/2000/svg">
-<path d="M15.4688 6.53125L16 6L15.4688 5.46875L11.2188 1.21875L10.6875 0.6875L9.625 1.75L10.1562 2.28125L13.125 5.25H0.75H0V6.75H0.75H13.125L10.1562 9.71875L9.625 10.25L10.6875 11.3125L11.2188 10.7812L15.4688 6.53125Z" fill="#0E0E0E"/>
```
[the remark this answers is not in the document]
Comment (Zmuse001) [reply app/assets/images/Icon.svg:2]: i don't know how to reverse deleting an image.

**Review by camckin10 — COMMENTED** [review]
Nice! 
A few things before its ready to merge: 
1) I left a code suggestion to remove the extra button
2) You will need to fix your merge conflicts 
3) You will also need to 're-request' an apprentice and tech lead review once you make the changes.

**Comment by ridwvn22** [comment]
Screenshot 2024-05-28 at 12 36 26 PM

It looks great!

**Review by ridwvn22 — APPROVED** [review]
looks good

**Review by edithemmings — COMMENTED** [review]
There are still way of being changes in here

**Comment by Zmuse001** [comment]
> There are still way of being changes in here

i took it out

**Review by crestu — APPROVED** [review]
looks good 👍

File: app/views/ways_of_being/index.html.erb [hunk 3]
```diff
@@ -2,9 +2,9 @@
     <div class="container bg-white h-screen w-64 rounded-lg">
         <h1 class="text-center font-bold lg:text-2xl py-4 border-b border-gray-600">Behaviors</h1>
         <ul class="px-4">
-            <% @ways_of_being.each do |way_of_being| %>
-            <li class="bg-white py-2 border-b border-gray-400 text-gray-400">
-                <%= way_of_being.name %>
+            <% @way_of_being.each do |way_of_being| %>
+            <li class="flex w-full justify-between bg-white py-2 border-b border-gray-400 text-gray-400">
+                <div><%= way_of_being.name %></div>
```
Comment (edithemmings) [inline app/views/ways_of_being/index.html.erb:7]: ^^^ The ways of being view is broken on your branch

**Review by edithemmings — APPROVED** [review]
Looks great!! Nice clean up from a tricky rebasing situation! 🎉🚀

**Review by crestu — APPROVED** [review]
nice work!

**Pull request merged**